src/kvstore/NebulaStore.cpp

```diff
@@ -203,9 +214,19 @@ void NebulaStore::addPart(GraphSpaceID spaceId, PartitionID partId) {
                                                    partId),
                                targetEngine.get(),
                                ioPool_,
-                               workers_));
-    // TODO: Need to pass in the peers
-    spaceIt->second->parts_[partId]->start({});
+                               workers_,
+                               flusher_.get()));
```

Comment:
Is there a problem that we config two data path like a and b, then there are two engines engine_a and engine_b, and engine_a has two parts 1 and 2, engine_b has 3. when call addPart with partId 1, then minIndex is engine_b, and the result cause that part 1 in two engine?

Reply:
Same part can't be added twice.

Reply:
> Same part can't be added twice.

is it possible to test this situation?

Reply:
Of course.

Reply:
When we find the target engine, we just check the minPartsNum, not check the part is in other engine?

Reply:
Please check addPart  function.

Reply:
when NebulaStore init, the hash map “spaceIt->second->parts_.find(partId)” is not empty?